back-end/routes/api.php

```diff
@@ -31,16 +32,27 @@
 });
 
 //------------------All-Private-Routes------------------//
-Route::group(['middleware' => 'auth:sanctum'], function () {
+Route::group([
+    'middleware' => 'auth:sanctum',
+    'prefix' => '/v1'
+], function () {
 
     // Admin Routes.
-    Route::resource('/v1/admin', AdminController::class);
+    Route::resource('/admin', AdminController::class);
 
     // User Routes.
-    Route::post('/v1/users/logout', [UsersController::class, 'logout']);
-    Route::resource('/v1/users', UsersController::class)
+    Route::post('/users/logout', [UsersController::class, 'logout']);
+    Route::resource('/users', UsersController::class)
         ->only(['index', 'show', 'update']);
 
     // Lessons Routes.
-    Route::resource('/v1/lessons', LessonsController::class);
+    Route::resource('/lessons', LessonsController::class);
+
+    // Follows Routes.
+    Route::resource('/follows', FollowsController::class);
+    Route::prefix('/follows')->group(function () {
+        Route::delete('/', [FollowsController::class, 'destroy']);
```

Comment:
- You have declared the resource above and then you again declare the delete here which should be included in the resource here.
- The exact route of destroy should be

```suggestion
        Route::delete('follows/{id}', [FollowsController::class, 'destroy']);
```
So in your destroy method, you dont need to use the `$request`.

Reply:
All the comments are noted and I'll fix them right away. Thank you, Sir Chev.